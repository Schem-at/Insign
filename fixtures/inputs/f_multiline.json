[
  { "pos": [0, 64, 0], "text": "@complex_region=rc([10,20,30],[40,50,60])\n#complex_region:metadata.type=\"storage\"\n#complex_region:metadata.capacity=1000\n#complex_region:metadata.active=true" }
]
