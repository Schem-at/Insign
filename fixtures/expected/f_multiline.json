{
  "complex_region": {
    "bounding_boxes": [
      [
        [
          10,
          84,
          30
        ],
        [
          40,
          114,
          60
        ]
      ]
    ],
    "metadata": {
      "metadata.active": true,
      "metadata.capacity": 1000,
      "metadata.type": "storage"
    }
  }
}
